src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx

```diff
@@ -128,10 +127,10 @@ class IntroductionPage extends React.Component {
             startText="Add or remove a dependent"
           />
           <div className="omb-info--container vads-u-padding-left--0">
-            <OMBInfo
-              resBurden={30}
-              ombNumber="2900-0043"
-              expDate="09/30/2021"
+            <va-omb-info
+              res-burden={30}
+              omb-number="2900-0043"
+              exp-date="09/30/2021"
```

Comment:
What is this date? it reads like a past expiration

Reply:
@tblackwe I'm not sure, a look through the code shows a few that would be past expiration and a few that haven't expired yet.

Reply:
definitely an expiration date. https://design.va.gov/storybook/?path=/docs/components-va-omb-info--children

suggest checking with design/other engineers to determine if this should be updated.

Reply:
orrrr.... We could just pull it off the 674 form.

> OMB Approved No. 2900-0049
> Respondent Burden: 15 minutes
> Expiration Date: 10/31/2024

https://www.vba.va.gov/pubs/forms/vba-21-674-are.pdf

Reply:
@tblackwe I updated the control number and it's ready for another review

Reply:
@tblackwe disregard that, it should stay at 0043, i was looking at the 674 form. pushed it back.